tcms/testruns/static/testruns/js/get.js

```diff
@@ -409,9 +409,26 @@ function renderTestExecutionRow(testExecution) {
     // remove from expanded list b/c data may have changed
     delete expandedExecutionIds[expandedExecutionIds.indexOf(testExecution.id)]
 
+    template.find('.change-status-button').click(function () {
+        const statusId = $(this).attr('data-status-id')
+
+        jsonRPC('TestExecution.update', [testExecution.id, {
+            'status': statusId,
```

Comment:
If there is a text entered inside the editor it is not added to the comments section when status buttons are clicked. This is a regression in behavior compared to the current implementation.